<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1026</width>
      <height>631</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_Ctrl_O"/>
    <addaction name="action_Ctrl_S"/>
    <addaction name="action_Ctrl_Alt_S"/>
    <addaction name="separator"/>
    <addaction name="action_Ctrl_Q"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionPreset"/>
    <addaction name="action_copy_param"/>
    <addaction name="separator"/>
    <addaction name="actionChangePsw"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模式</string>
    </property>
    <addaction name="action_mode_edit"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_T"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>新建 Ctrl+N</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Ctrl_O">
   <property name="text">
    <string>打开 Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Ctrl_S">
   <property name="text">
    <string>保存 Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Ctrl_Alt_S">
   <property name="text">
    <string>另存为... Ctrl+Alt+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_Ctrl_Q">
   <property name="text">
    <string>退出 Ctrl+Q</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreset">
   <property name="text">
    <string>预设 Ctrl+P</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_copy_param">
   <property name="text">
    <string>一對多参数复制 Ctrl+J</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionChangePsw">
   <property name="text">
    <string>修改密碼</string>
   </property>
  </action>
  <action name="action_mode_control">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>控制</string>
   </property>
  </action>
  <action name="action_mode_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编辑OID</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
